org.Cataclysm.DDA.Tiles.yml

```diff
@@ -0,0 +1,54 @@
+id: "org.Cataclysm.DDA.Tiles"
+runtime: "org.freedesktop.Platform"
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: /app/bin/launch-cataclysm-dda
+finish-args: 
+finish-args: 
+  [
+    "--filesystem=xdg-config/cataclysm-dda:create",
+    "--socket=pulseaudio",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--share=ipc",
+    "--device=dri"
+  ]
+
+modules:
+  - name: cataclysm-dda
+    buildsystem: simple
+    build-commands:
+      - tar -xvf cdda-linux-tiles-x64-2023-03-01-0054.tar.gz -C /app
+      - rm cdda-linux-tiles-x64-2023-03-01-0054.tar.gz
+      - mkdir -p /app/bin
+      - |
+        cat > /app/bin/launch-cataclysm-dda << EOF
```

Comment:
make this either a separate file. or a `script` source. Not inline in the build commands.